bigbluebutton-html5/package.json

```diff
@@ -14,7 +14,6 @@
     "precommit": "lint-staged"
```

Comment:
We need to get rid of this line. It was properly removed here: https://github.com/bigbluebutton/bigbluebutton/pull/6346/commits/44c7e5c10abfcb1f2be897b7ae29be57fbbeeff1#diff-b9183517a2d810804e342afa2421896dL14, but it got back through this merge commit: https://github.com/bigbluebutton/bigbluebutton/pull/6346/commits/462f20d9bbab2626c53bdb7612b78e006f385d10.